### ISSUE TITLE
Integrate Prometheus logging into DPv1

### ISSUE THREAD

**Issue by RangerMauve** [hyphacoop/distributed-press-organizing#60]
🎟️ **Re-ticketed from:** #  
📅 **Due date:** N/A  
🎯 **Success criteria:** Initial promethius integration is in fastify in v1

## Task Summary

Jacky made progress on integrating [fastify-metrics](https://gitlab.com/m03geek/fastify-metrics) to add prometheus logging, it's currently blocked on getting some typescript types working.

We'll split this out to track progress on fixing the types.

After we get the initial integration in we can look at tracking more logs like the p2p things.

...

## To Do

- [ ] Figure out what's missing in types
- [ ] Send a PR or generate a local d.ts override
- [ ] Integrate fastify-metrics
- [ ] PR into v1
- [ ] Merge!

**Comment by jackyzha0** [comment]
Silly mistake on my end, accidentally removed `"esModuleInterop": true` from `tsconfig.json` which caused `import client from "prom-client"` to not be transpiled to `import * as client from "prom-client"`

Changes are in https://github.com/hyphacoop/api.distributed.press/pull/31/

https://github.com/hyphacoop/api.distributed.press/pull/31/commits/295765a1ccd8882f4fb11ec240d0d6b6d6c07df9#diff-188928d8ce5c2cceeb0ac66c24516158b0656dc4fb6be31eaccceeeee29a699b
